tests/TestCase/Shell/CacheShellTest.php

```diff
@@ -34,7 +34,7 @@ public function setUp()
         parent::setUp();
         $this->io = $this->getMockBuilder('Cake\Console\ConsoleIo')->getMock();
         $this->shell = new CacheShell($this->io);
-        Cache::config('test', ['engine' => 'File', 'path' => TMP]);
+        Cache::config('test', ['engine' => 'File', 'path' => TMP . 'cache']);
```

Comment:
Sorry, I misunderstood.
It seems to me that it is automatically added (otherwise it would fail the test, no?).
However, if desired, we could be added it for completeness.